src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
+/* eslint-disable prefer-promise-reject-errors */
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const div = document.createElement('div');
+
+div.classList.add('message');
```

Comment:
move all logic with creating the message to some function and just call that function in your then and catch